src/components/tiptap/notificationWidget/NotificationWidget.tsx

```diff
@@ -51,6 +55,13 @@ export const NotificationWidget = () => {
                         key={key}
                         name={`Pay ${invoiceCount} invoices`}
                         route={PortalRoutes.Billing}
+                        display={
+                          pathname.includes('client-preview')
+                            ? Number(invoiceCount) > 0
+                            : appState?.appState.readOnly
+                              ? Number(invoiceCount) > 0
+                              : true
```

Comment:
Since this logic is repeated in three places and contains nested ternaries, please consider moving it to a reusable util function

Reply:
fixed in latest commit